lib/gateway/docker-compose.gateway.yaml

```diff
@@ -104,6 +105,7 @@ services:
       - ${BENTOV2_GATEWAY_CERTS_DIR}:${BENTOV2_GATEWAY_INTERNAL_CERTS_DIR}:ro
       - ${PWD}/lib/gateway/services:/gateway/services:ro
       - ${PWD}/lib/gateway/public_services:/gateway/public_services:ro
+      - /var/log:/var/log
```

Comment:
this shouldn't be bound to the system `/var/log`, since this can easily clash with other logs on the host – this should instead be set up as a volume for the gateway logs, or not bound at all, I think.